site.yml

```diff
@@ -157,6 +157,14 @@
       tags: ['ceph', 'ceph-osd']
   environment: "{{ env_vars|default({}) }}"
 
+- name: ceph update
+  hosts: ceph_monitors:ceph_osds:controller:compute
```

Comment:
I keep reading the `ceph-update` role and I just don't see any template tasks in it. I can't find a single task in this role that would run on compute or controllers, other than just a ceph mons restart.  What am I missing?